backend/src/main/java/ca/bc/gov/app/service/client/ClientService.java

```diff
@@ -446,6 +447,7 @@ private Mono<ClientDetailsDto> buildAddress(
                 loadProvince(address.country().value(), address.province().value())
                     .map(address::withProvince)
             )
+            .sort(Comparator.comparing(ClientAddressDto::locationName).reversed())
```

Comment:
Not necessarily. As long as the frontend association is done, the name doesn't matter. But, as the object always returns mailing and delivery addresses, and we derive the name from the property name, by sorting it like this, we make sure that the mailing always comes first.

Reply:
I think the locationName matters, I think that's the cause of the reported issue.
The first address returned by the API has to be called "Mailing address", otherwise we get the issue.
Do you know if there is a risk of having some other address included?
If it includes another address alphabetically greater than "Mailing address", we would get the issue again.

Reply:
No, no other addresses will be included (as far as we know). If the BC Registry includes a new address, we will handle it accordingly.